content/blog/2023-05-16-comparison-of-ethereum-classic-and-ethereum-monetary-policies/index.md

```diff
@@ -0,0 +1,146 @@
+
+This means that the staking industrial complex will perpetually get richer, creating mega-validators, while the rest of the economy will be constantly starved of money to pay for fees.
+
+While there will alway be some coin to pay for fees, there is a giant constant wealth transfer from the economy to the staking cartel.
+
+## Digital Gold
+
+ETC is digital gold and ETH is not.
+
+ETC is digital gold because it is a proof of work blockchain and the purpose of that consensus mechanism is to make it as costly to create ETC as it is costly to produce gold in the real world.
+
+The way it works is that miners in the blockchain create blocks of transactions and to stamp them cryptographically they need to do a lot of computational work, which uses a lot of electricity. Then, when the blocks are accepted by the rest of the network, the miners are credited newly issued coins by the algorithm.
+
+The above system guarantees that ETC will be sound money as gold was in the old days, hence the term “digital gold”.
+
+ETH is not digital gold because it is a proof of stake blockchain and validators do not have to do any work to produce blocks or earn rewards. They just receive transactions, package them into batches they call blocks to steal some lure from proof of work blockchains, and they vote on them to affirm their validity.
+
+The above means that to produce the coin in Ethereum is practically costless, thus the currency cannot be compared to gold.
+
+The best comparison that may be made for ETH is that it is very similar to fiat money. Fiat money is produced by central banks with no cost whatsoever, thus ETH may be called community fiat money as it is printed by a private community rather than a government. 
+
+## Algorithmic
+
+ETC’s monetary policy is algorithmic because in 2017 it migrated to a schedule that is set in stone as it establishes the whole future distribution in the software in the form of a supply algorithm.
+
+The supply algorithm is a set of simple and transparent mathematical rules that establish what number of coins will be paid to miners per block, the frequency of the payments, and a schedule that will decrease these payments as time passes by.
+
+ETH’s monetary policy is manual, not algorithmic, because it has been changed several times and will likely change many times more in the future given that proof of stake tends to centralization.
```

Comment:
Line 57: Usage
```suggestion
ETH’s monetary policy is manual, not algorithmic, because it has been changed several times and will likely change many more times in the future given that proof of stake tends towards centralization.
```